pkg/tsdb/mssql/macros.go

```diff
@@ -90,7 +90,7 @@ func (m *msSqlMacroEngine) evaluateMacro(name string, args []string) (string, er
 	case "__timeGroupAlias":
 		tg, err := m.evaluateMacro("__timeGroup", args)
 		if err == nil {
-			return tg + " AS [time]", err
+			return tg + " AS [time]", nil
```

Comment:
`err` is known to be `nil` here, so it's better to be explicit about there being no error.